### ISSUE TITLE
Ignoring double quotes on Sublime Text Key Map file

### ISSUE THREAD

**Issue by evandrocoan** [facelessuser/BracketHighlighter#392]
### Current behavior

Here the syntax is Sublime Text Key Map and the quotes are not underlined on line 3.

![image](https://cloud.githubusercontent.com/assets/5332158/20674946/d2f68754-b571-11e6-8203-70f2d6812d93.png)

### Expected behavior

Here the syntax is C++ and the quotes are underlined on line 4.

![image](https://cloud.githubusercontent.com/assets/5332158/20675026/0fb10b2e-b572-11e6-8c4f-41a62e97c7ab.png)

## Support Info

```
- ST ver.: 3126
- Platform: windows
- Arch: x32
- Plugin ver.: 2.23.1
- Install via PC: True
- mdpopups ver.: 1.11.0
- backrefs ver.: 1.0.post1
- markdown ver.: 2.6.6
- pygments ver.: 2.1a0
- jinja2 ver.: 2.8
```

## Steps to Reproduce Issue

Create the file `Default.sublime-keymap` using this contents and the syntax Sublime Text Key Map, and put the cursor/caret inside some double quotes.
```javascript
[
  // open a docblock with enter
  { "keys": ["enter"], "command": "jsdocs",
    "context": [
      { "key": "setting.auto_indent",   "operator": "equal",          "operand": true,                        "match_all": true },
      { "key": "selection_empty",       "operator": "equal",          "operand": true,                        "match_all": true },
      { "key": "auto_complete_visible", "operator": "equal",          "operand": false,                       "match_all": true },
      { "key": "preceding_text",        "operator": "regex_contains", "operand": "^\\s*(\\/\\*|###)[*!]\\s*$", "match_all": true }
    ]
  },
]
```

**Comment by r-stein** [comment]
This is essentially an issue of the PackageDev syntax definition https://github.com/SublimeText/PackageDev/issues/43

**Comment by facelessuser** [comment]
Thanks @r-stein, you are correct.  PackageDev chooses a different way to style their strings.  If they choose to do things differently than how the current string rules expect, you will not get string matching in those syntaxes.

My stance on bracket rule support is that they are supported by the community (unless I have a vested interest in the specific rule).  I don't use PackageDev, so there is no motivation for me to dig in and come up with a PackageDev specific quote highlighting rules, but I would gladly accept them via a pull request.  I am even available to answer questions if you or someone else attempts a pull and gets stuck on something.

With all I have going on, it is quite time consuming looking into the nuances of every syntax highlighter package and obscure language that I don't use.  I will gladly add them via pulls, but I need the community to pitch in to help with these kind of things so I can focus on actual bugs and features of this plugin and others (which I always have coming in :wink:).

In short, PackageDev needs to either conform to the current string logic, or there will need to be a pull request to add PackageDev string logic.  Or alternatively, don't worry about quotes in PackageDev :smile:.
